app/indexers/default_work_indexer.rb

```diff
@@ -17,13 +17,19 @@ def generate_solr_document
       license_labels = ScholarsArchive::LicenseService.new.all_labels(object.license)
       language_labels = ScholarsArchive::LanguageService.new.all_labels(object.language)
       peerreviewed_label = ScholarsArchive::PeerreviewedService.new.all_labels(object.peerreviewed)
+      academic_affiliation_labels = ScholarsArchive::TriplePoweredService.new.fetch_top_label(object.academic_affiliation, parse_date:true)
```

Comment:
...and finally, there are a test or two that would need to respect this data structure as well as a has_url_validator.rb and triple_powered_form.rb